### DISCUSSION TITLE
Version 1.0

### DISCUSSION THREAD

**Discussion by AlexKluev1989** [arrow-kt/arrow#2353] [General]
Hello folks!

Do you have any milestones and when do you plan to release production readly version 1.0 and what should be implemented/fixed to achieve this?

**Comment by nomisRev** [comment]
Hey @AlexKluev1989,

I'm not sure if you saw the events of last days where we released 0.12.x & 0.13.x. This paves the way to Arrow 1.0 which will come soon, some more details here in the release post. https://www.47deg.com/blog/arrow-0.13.0-release/

TL;DR 1.0.0-SNAPSHOT is already available and soon a 1.0.0-MILESTONE1 will be released. 0.13.0 is the last version before 1.0.0 and we'll be making it MPP in the next weeks.

When 0.13.0 has seen some more usage, and we've were able to collect some more feedback we'll be ready for 1.0.0 MILESTONE releases and 1.0.0 will be released soon.

**Comment by kyay10** [comment]
Closed since we're already at version 2!
